### ISSUE TITLE
Write a guide

### ISSUE THREAD

**Issue by the-ethan-hunt** [the-ethan-hunt/B.E.N.J.I.#119]
although we have an `activities.md` folder, now B.E.N.J.I. is expanding and we need to have a help guide either on wiki, Github Gist or just markdown pages. Basic example guide pages include
-  How to install on various OS
- How to use the different features
- FAQs
-  Bug reporting
P.S. The issue might seem very easy but requires **heavy documentation with proper language**. Please **don't take this lightly**

**Comment by Shristy2404** [comment]
I can attempt this. One problem though that I have Windows and Linux so I'll update for those, I won't be able to add much for Mac than what already exists. Can I take this up?

**Comment by the-ethan-hunt** [comment]
@Shristy2404 Please explain how you would be working on this. Elaborately

**Comment by HemabhKamboj** [comment]
I can work on this under GSSoC 19, for Windows and Linux 

Markdown pages will be a good option, but also we can think to create a wiki on GH pages, will look better with some CSS. I think GitHub gists won't be a good option.

- For installation on OS, A step by step guide, from cloning to running scripts and screenshots (or gif of the whole installation or video guide with narrative steps uploaded on youtube)

- Solutions of common problems encountered during installation of on each OS ( in the installation guide only, but not in video)

- To demonstrate the use of different features, expand and elaborate activities.md

- Will try to write FAQ's myself and also create a thread on slack for more help on these. A different page on FAQs will be good.

-For bug reporting, first asking if the user has gone through FAQ's and issues on GitHub, then a standard way to follow (maybe email or google form, will not suggest to directly create an issue)

**Comment by angelapwen** [comment]
I can do this for MacOS and make sure it matched the Windows and Linux installation formats and such 😊 also under GSSoC 19

**Comment by the-ethan-hunt** [comment]
For Windows: Assigned to @HemabhKamboj 
For Mac OS: Assigned to @angelapwen

**Comment by HemabhKamboj** [comment]
For linux also.

**Comment by angelapwen** [comment]
Hi Dhruv, I was looking at https://github.com/the-ethan-hunt/B.E.N.J.I./community and wondering if you would want us to create pages for Contributing.md and the other two, for Pull Request and Issue Templates as well? 

If you want us to create them, would you want these within the FAQ, or separate markdown files in the main repo?

**Comment by the-ethan-hunt** [comment]
@angelapwen sounds nice, but you already have an issue assigned. How about you work on that and then the templates? :smile:

**Comment by angelapwen** [comment]
@the-ethan-hunt Yes, sure! I'll work on the other issue. Maybe @HemabhKamboj can submit the PR for Linux/Windows, and once that one is merged I will follow his format to submit a separate PR for Mac. And after the other issue, the templates. ✔️
